cypress-io/cypress
# fix: prevent vite from crashing where there are no support files or specs found

When there is no values, in the optimization array,
supportFile can be a boolean. the final optimized entry passed to
fast-glog is then `[false]` whic crashes cypress.

This protects against the no-spec issue

closes #17373

## How to reproduce

- create a vite app (`yarn create vite-app`)
- create cypress component testings around it (`yarn create cypress-tests`) 
- in the `cypress.json` set `supportFile` to `false` and make sure there is no spec files.

Here is the crash.
